api/cpp/include/slint-platform.h

```diff
@@ -140,6 +140,7 @@ class WindowAdapter
 class Platform
 {
 public:
+    using Clipboard = cbindgen_private::Clipboard;
```

Comment:
This will look a little strange in the documentation (`cargo xtask cppdocs`, then open `target/cppdocs/html/index.html`. I think the enum should be placed in `public_exported_types` in `cbindgen.rs` and then used as top-level.

The other alternative would be to go back to duplicating it.

API wise I don't feel too strongly. My feeling is that the enum might be useful for public clipboard access API later, too (so top-level).

Reply:
I feel like this enum should stay in `platform` namespace for now.
And while it would be better to have it auto-generated, I'm fine having it duplicated as it's easier.

Reply:
IMHO this is the last problem that prevent this to be merged:  It will not look good in the docs.

If you don't have time to fix it yourself, i can fix it myself.

Reply:
@ogoffart I made an attempt at swapping it to public export but it didn't immediately work, and I am indeed much more busy as of the last week so I would appreciate if you did a fixup after this PR, thanks :)